$NetBSD: patch-tools_boost__install_boost-install-dirs.jam,v 1.1 2026/02/08 22:36:16 wiz Exp $

For some reason, without this patch, "stage-includedir" is set the BOOST_ROOT
variable corresponding to the staging directory (WRKSRC/stage) without the
"include" subdir. This messes up the cmake *-config.cmake files generated
during the staged build.

--- tools/boost_install/boost-install-dirs.jam.orig	2025-12-03 13:46:48.000000000 +0000
+++ tools/boost_install/boost-install-dirs.jam
@@ -69,10 +69,6 @@ rule stage-includedir
         ld = [ path.root [ path.make $(ld) ] [ path.pwd ] ] ;
         return $(ld) ;
     }
-    else if [ modules.peek boostcpp : BOOST_ROOT ]
-    {
-        return [ path.make [ modules.peek boostcpp : BOOST_ROOT ] ] ;
-    }
     else
     {
         local stagedir = [ stagedir ] ;
